tests/test_callback.py

```diff
@@ -0,0 +1,37 @@
+import os
+import re
+import ansible_runner
+
+
+def run_playbook(tmpdir):
+    os.environ['ANSIBLE_STDOUT_CALLBACK'] = "foreman"
```

Comment:
```suggestion
    os.environ['ANSIBLE_STDOUT_CALLBACK'] = "foreman"
    os.environ['ANSIBLE_CALLBACK_WHITELIST'] = "foreman"
```

Otherwise it will never allow that callback